packages/components/psammead-image-placeholder/src/index.stories.jsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import { storiesOf } from '@storybook/react';
```

Comment:
Why do we no longer need the eslint-disable?

Reply:
That `eslint-disable` was needed in Simorgh, but isn't in Psammead, since there is a global ignore of this rule.